modules/lazyload/assets/js/frontend.js

```diff
@@ -15,7 +16,7 @@ document.addEventListener( 'DOMContentLoaded', function() {
                 lazyloadBackgroundObserver.unobserve( element );
             }
         } );
-    } );
+    }, options );
```

Comment:
Consider moving this entire script into an inline script for better performance.